renderer/components/cropper/overlay.js

```diff
@@ -0,0 +1,137 @@
+// Packages
+import electron from 'electron';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// Containers
+import {
+  connect,
+  CursorContainer,
+  CropperContainer,
+  ActionBarContainer
+} from '../../containers';
+
+// Utilities
+import {getResizingCursor} from './handles';
+
+class Overlay extends React.Component {
+  remote = electron.remote || false
```

Comment:
I don't understand this check. When would `electron.remote` not be defined?

Reply:
This is standard practice for using Next.js in electron, as seen in Zeit's own [desktop app](https://github.com/zeit/now-desktop/blob/canary/renderer/components/feed/switcher.js#L43-L53).

Reply:
That makes for some really awkward code... Probably because of server-rendering, which we don't really need with the Electron app. Is there any way to turn off server-rendering?

Reply:
Yeah, exactly. I'm not sure haha, I think server rendering is a major part of Next.js